app/controllers/builder_controller.rb

```diff
@@ -0,0 +1,85 @@
+class BuilderController < ApplicationController
+  STRIPE_CHARGE_DESCRIPTION = 'GramGram Photo'.freeze
+  RECIPIENT_PARAMS = %i[
+    name
+    address_line1
+    address_line2
+    address_city
+    address_state
+    address_zip
+  ].freeze
+
+  def new
+    photo = current_user.photos.find(params[:photo_id])
+    postcard = Postcard.create(photo: photo)
+
+    redirect_to build_caption_path(postcard)
+  end
+
+  def caption
+    @postcard = postcard
```

Comment:
There are 3 actions that make this assignment. Would it be better to have this be a `before_action` method instead? This is in nit territory, but I'm curious what you think.

Reply:
Totally agree! That is a good call